hack/run-wsu-ci-e2e-test.sh

```diff
@@ -47,6 +47,6 @@ oc patch network.operator cluster --type=merge -p '{"spec":{"defaultNetwork":{"o
 
 # Run the test suite
 cd $TEST_DIR
-GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on CLUSTER_ADDR=$CLUSTER_ADDR WSU_PATH=$WMCO_ROOT/tools/ansible/tasks/wsu/main.yaml go test -v -vmCreds="$VM_CREDS" $SKIP_VM_SETUP -timeout 30m .
+GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on CLUSTER_ADDR=$CLUSTER_ADDR WSU_PATH=$WMCO_ROOT/tools/ansible/tasks/wsu/main.yaml go test -v -vmCreds="$VM_CREDS" $SKIP_VM_SETUP -timeout 60m .
```

Comment:
why to increase timeout? Is it because of the running multiple times?

Reply:
Yep! I put it in my commit message. Doubling the timeout since we are doubling the tests. There is some setup that isnt repeated, but IMO its better to give too much time than too little. I run into test timeouts locally too much :)

Reply:
I failed to update the MR message, did that now